lib/eco/eco.py

```diff
@@ -1338,6 +1529,119 @@ def run(self):
                         line = "  " + line[9:]
                 self.emit(self.signal, line.rstrip())
 
+class Debugger(QObject):
+    def __init__(self, window):
```

Comment:
Because the eco.py file is already very long, it would be nice to factor this class out into its own file.